strawberry_django_plus/permissions.py

```diff
@@ -37,7 +39,10 @@
 from .types import OperationInfo, OperationMessage
 from .utils import aio, resolvers
 from .utils.query import filter_for_user
-from .utils.typing import UserType
+
+with contextlib.suppress(ImportError, RuntimeError):
+    from django.contrib.auth.models import AbstractUser
```

Comment:
Shouldn't this be in a `try/except` with `AbstractUser = None` in case of errors?

Reply:
pyupgrade, but fixed this